packages/react-components/src/icons/src/Icon.tsx

```diff
@@ -49,7 +62,7 @@ export function InnerIcon(props) {
                         "o-ui-icon",
                         disabled && "disabled",
                         normalizeSize(size),
-                        color ? `color-${color}`: ""
+                        color ? `color-${color}` : ""
```

Comment:
I don't think this is used. I think you added this in the dark mode branch @fraincs can you confirm?

Reply:
This can be removed